iina/MainWindowController.swift

```diff
@@ -2114,7 +2114,7 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
     if player.info.isNetworkResource {
       bufferIndicatorView.isHidden = false
       bufferSpin.startAnimation(nil)
-      bufferProgressLabel.stringValue = "Opening stream..."
+      bufferProgressLabel.stringValue = NSLocalizedString("main.opening_stream_indicator", comment: "Opening stream…")
```

Comment:
See https://github.com/lhc70000/iina/pull/1949#discussion_r215097657

Reply:
OK. I'll do that.